dev/build/windows/makecoq_mingw.sh

```diff
@@ -1696,8 +1746,10 @@ function make_addon_menhirlib {
 # CompCert
 
 function make_addon_compcert {
+  installer_addon_dependency_beg compcert
   make_menhir
   make_addon_menhirlib
+  installer_addon_dependency_end
```

Comment:
For some reason, I can select CompCert and it won't select menhir. Am I right to think that this is not expected?

Reply:
There is a module "make_menhir" and a module "make_addon_menhir". The first just compiles menhir and makes it available at build time. CompCert needs menhir for building. The addon  actually installs menhir with Coq.

As far as I can tell for proving code correct against CompCert (like VST does) one does (or might) need menhirlib (to reason about the generated parser files), but one does not need menhir. At least I think some of the .vo files installed with CompCert depend on menhirlib.

So this works as intended. A dependency would be generated if compcert would call "make_addon_menhir" instead of "make_menhir", but as far as I understand this, this dependency is not required.

Reply:
OK perfect then.